### DISCUSSION TITLE
When will "Prefer Smaller Files" return?

### DISCUSSION THREAD

**Discussion by tr1p0p** [BrenekH/encodarr#129] [General]
Hi !

Glad to see ths project still run, i was just wondering, when did you plan to put this option "don't encode if not reduce file size" ? It's kind of really important for my managing of my video library, it's too bad because i cannot use encodarr in this state because it re-encode all my video library despite the fact a lots of it don't need it.

I saw you seem alone on this project, and it could be a hard work, thanks again for this tools, and don't find anything close to what it offers, so again thanks to you.

**Comment by BrenekH** [comment]
The eventual plan is to re-add this functionality using the plugin system that is still in the planning stages. I was hoping that it would be done by now, but I've been fairly busy with school and other projects. Luckily, we break for the holidays in about a week so hopefully I can get some development time in soon.

**Reply by tr1p0p** [reply]
Ok thanks for the answer ! Great to know this option will go back, i understand you're quite busy, enjoy yours holidays !
